### PR TITLE
[sonic-acl.yang] Add new ACL key BTH_OPCODE and AETH_SYNDROME

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### Why I did it

Add new ACL key BTH_OPCODE and AETH_SYNDROME

#### How I did it

Add new ACL key BTH_OPCODE and AETH_SYNDROME

#### How to verify it

1. manual test
2. unit test

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Review by stepanblyschak — APPROVED** [review]

**Review by keboliu — APPROVED** [review]

File: src/sonic-yang-models/yang-templates/sonic-acl.yang.j2 [hunk 1]
```diff
@@ -268,6 +268,18 @@ module sonic-acl {
 						range "0..1";
 					}
 				}
+
+				leaf BTH_OPCODE {
```
Comment (ganglyu) [inline src/sonic-yang-models/yang-templates/sonic-acl.yang.j2:272]: Please add description for BTH_OPCODE and AETH_SYNDROME

Comment (Junchao-Mellanox) [reply src/sonic-yang-models/yang-templates/sonic-acl.yang.j2:272]: Added

**Review by ganglyu — APPROVED** [review]

**Comment by Junchao-Mellanox** [comment]
Hi @qiluo-msft , could you please help review and merge this?

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**